<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComputerDialog</class>
 <widget class="QDialog" name="addComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="InputComputerName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="addComputerPicture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputBuiltYear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Build year</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeDropDown">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mechanical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vacuum</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transistors</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Digital</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other...</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LinkToScientist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="C_add_ok">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioWasBuilt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Was built?</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleComputerAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Add new computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputComputerType">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InvalidInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Invalid input</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
